CHANGES.md

```diff
@@ -1,38 +1,97 @@
+# 2.17.0 - 26 Sep 2024
+
+## Features
+
+* optionally install Scenario with `ops[testing]` and expose the names in ops.testing in (#1381)
+* add a JujuVersion property for Pebble log forwarding to Loki in (#1370)
```

Comment:
We strip out the "in" at the end as well, e.g.

```suggestion
* add a JujuVersion property for Pebble log forwarding to Loki (#1370)
```

Same comment for all the bullets.

Tbh, I think the link/don't link aspect could be reconsidered. Feel free to bring it up in a team meeting.